src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -31,7 +31,7 @@ internal static partial class Libraries
         internal const string Wldap32 = "wldap32.dll";
         internal const string Ws2_32 = "ws2_32.dll";
         internal const string Wtsapi32 = "wtsapi32.dll";
-        internal const string CompressionNative = "clrcompression";
+        internal const string CompressionNative = "System.IO.Compression.Native";
```

Comment:
This can be move to `src\libraries\Common\src\Interop\Interop.Libraries.cs` now

Reply:
It would miss the `lib` prefix on Linux and incur an extra probe: https://github.com/dotnet/runtime/pull/33236#issuecomment-595276121?

Reply:
(unless it is ok to use `#if TARGET_UNIX` in this case 🙂)

Reply:
This string is defined as `libSystem.IO.Compression.Native` on Unix.

As I looked around it appears that `libSomething.so` on Unix is typically matched with `Something.dll` on Windows, so I assume the difference here is expected as well.

Reply:
Actually - GlobalizationNative is defined `internal const string GlobalizationNative = "libSystem.Globalization.Native";`  regardless of platform.

That is a bit different case, since GlobalizationNative is never a separate .dll file. Otherwise it would need to match the file name and on Windows would not have `lib` prefix.

@jkotas @stephentoub - How strong is the rule that dlls on Windows do not have `lib` prefix? 
Perhaps both Compression and Globalization native .so/.dll should have the `lib` regardless of platform?

Reply:
Alternatively GlobalizationNative should probably drop the `lib` prefix on Windows.

Reply:
I think it looks more natural without the lib prefix on Windows. We are not going to rename coreclr to libcoreclr on Windows either.

Independent on what we choose, both Compression and Globalization should be done the same way.

Reply:
So, for consistency with other dlls will not use prefix `lib` on Windows. 
Also made a corresponding change to the naming of Globalization. (no `lib` prefix on Windows)